### ISSUE TITLE
Suggestion - 'show (all) areas' command for Pneumatic Helmet Debugging

### ISSUE THREAD

**Issue by wrincewind** [TeamPneumatic/pnc-repressurized#197]
Just a nice Quality of Life option - when drone debugging, clicking on an area should show it, like clicking the 'show area' button in the Programmer. For bonus points, perhaps the area names (if they're being turned into variables) could hover above them, or possibly a 'press Y to view in debugger' option that'd open up the Drone Debug screen with all instances of that area highlighted in the code?  

A 'show/hide all areas' button on the debug screen would also be nice for this, just to make sure all coordinates are appearing as they should - as it is, i don't think there's any other way to see coordinate variables changing in real time without keeping an eye out for the red particles when the program hits that particular area.

**Comment by desht** [comment]
Yeah, there's room for improvement here for sure.  I'll see what can be added to make debugging more user-friendly.

**Comment by wrincewind** [comment]
While we're on the topic, a 'hide all areas' button would be useful, as would a 'jump to current command' button (maybe a 'follow' one that moves the screen around as the program jumps from command to command?) - and also a 'step' command, or even the possibility of adding breakpoints... Maybe a *little* advanced, but i'd love to be able to pause a drone on a certain command, see what the results of the previous commands are... 

Maybe even a way to view the drone's inventory (fluid, RF, and items)?

Oh, and speaking of inventory, the debug response from a successful 'pick up whitelisted item from a chest' is a little obtuse. If it finds the item, it'll say something like '168 items did not match the filter', but if it doesn't find the item it'll say '169 items did not match the filter' and continue regardless.  Needless to say this gave me some trouble when I was autocrafting and didn't realise i'd run out of an ingredient. I suppose I should do some checks with 'conditional: drone inventory' to ensure I've exactly the right items before doing a craft...

Sorry if this is a little rambly, i'm passing the time at work :P

**Comment by desht** [comment]
Build 262 (0.8.0 prerelease) from http://jenkins.k-4u.nl/job/PneumaticCraft-Repressurized/ contains some of what you're looking for:
* Areas can be shown/hidden by right-clicking the widget in the debug screen
* Buttons to jump to Start & current widget added
* Checkbox to continually follow current widget

I'm looking into some of the other stuff, but you have the easy bits now :)

**Comment by wrincewind** [comment]
Hey, thanks! :)
